### PR TITLE
FAT-1707 C350929 Verify Bulk Edit app - landing page

### PR DESCRIPTION
**Pull request by MaksimTerekhov**

### CODE REVIEW

**Review by TetianaParanich — APPROVED** [review]

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 1]
```diff
@@ -9,20 +9,119 @@ import {
...
+      Button('Reset all').click(),
+      Button('Identifier').click()
+    ]);
+  },
+
   verifyCsvViewPermission() {
     cy.expect([
       RadioButton('Inventory - items').has({ disabled: true }),
       Select('Record identifier').has({ disabled: true }),
       Button('or choose file').has({ disabled: true }),
```
[the remark this answers is not in the document]
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js]: done

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 2]
```diff
@@ -94,7 +197,7 @@ export default {
   },
 
   verifyActionsAfterConductedCSVUploading(errors = true) {
-    cy.do(Button('Actions').click());
+    cy.do(actions.click());
     cy.expect([
       Button('Download matched records (CSV)').exists(),
```
[the remark this answers is not in the document]
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:204]: done

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 3]
```diff
@@ -106,7 +209,7 @@ export default {
   },
 
   verifyActionsAfterConductedInAppUploading(errors = true) {
-    cy.do(Button('Actions').click());
+    cy.do(actions.click());
     cy.expect([
       Button('Download matched records (CSV)').exists(),
       Button('Start bulk edit').exists(),
```
[the remark this answers is not in the document]
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js:217]: done

File: cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js [hunk 4]
```diff
@@ -9,20 +9,119 @@ import {
...
+    cy.do([
+      Button('Reset all').click(),
+      Button('Identifier').click()
+    ]);
+  },
+
   verifyCsvViewPermission() {
     cy.expect([
       RadioButton('Inventory - items').has({ disabled: true }),
       Select('Record identifier').has({ disabled: true }),
```
[the remark this answers is not in the document]
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js]: done

**Review by kimkate303 — APPROVED** [review]

**Pull request merged**